cloud/docker/docker.py

```diff
@@ -1467,6 +1471,11 @@ def get_deployed_containers(self):
                 name_list = container.get('Names')
                 if name_list is None:
                     name_list = []
+                # When using docker swarm, our container names are of the format /node/container-name, so we need to
+                # convert these to just /container-name.
+                for index, item in enumerate(name_list):
+                    if '/' in item[1:]:
```

Comment:
:-1: 
:warning: some containers have a name like : /xxxx/yyyy even if it's not running in a swarm environment.
For example : ```[u'/etcd', u'/skydns/etcd']```

This bug is fixed in the PR #3158